shadows/framework/src/main/java/org/robolectric/shadows/ImageUtil.java

```diff
@@ -66,22 +67,31 @@ static RobolectricBufferedImage getImageFromStream(InputStream is) {
       initialized = true;
     }
 
+    String format = null;
     try {
       ImageInputStream imageStream = createImageInputStream(is);
       Iterator<ImageReader> readers = ImageIO.getImageReaders(imageStream);
-      if (!readers.hasNext()) return null;
+      if (!readers.hasNext()) {
+        return null;
+      }
 
       ImageReader reader = readers.next();
       try {
         reader.setInput(imageStream);
-        String format = reader.getFormatName();
+        format = reader.getFormatName();
         int minIndex = reader.getMinIndex();
         BufferedImage image = reader.read(minIndex);
         return RobolectricBufferedImage.create(image, ("image/" + format).toLowerCase());
       } finally {
         reader.dispose();
       }
     } catch (IOException e) {
+      if (FORMAT_NAME_PNG.equalsIgnoreCase(format) && e instanceof IIOException) {
+        System.err.println(
```

Comment:
Is there any way we could give a hint as to which PNG file is corrupt? Maybe the methods from BitmapFactory could pass some kind of information about which PNG file is being processed? That way users will at least have some details about the PNG file that is being processed.

Reply:
I will check it. It's better if we can print PNG file name.